apps/hash-graph/lib/authorization/src/zanzibar/api.rs

```diff
@@ -168,7 +168,11 @@ where
         entity: EntityId,
     ) -> Result<Zookie<'static>, ModifyRelationError> {
         Ok(match scope {
-            VisibilityScope::Public => unimplemented!(),
+            VisibilityScope::Public => {
+                self.backend
+                    .create_relations([(entity, EntityRelation::DirectOwner, PublicAccess)])
+                    .await
+            }
```

Comment:
This looks like it's handling adding 'anyone' as an entity owner – is that right, and if so do we need it? Same applies to deleting an owner.

Reply:
Yes, it's just for consistency. Previously if this happened the code would have panicked, now it would fail with an error as the SpiceDB schema does not allow this. Ideally this cannot happen at all and I'm going to change this later but for now it's the easiest way to achieve this.

Reply:
Okay cool, as long as we change it later